src/building/type.h

```diff
@@ -168,8 +168,9 @@ typedef enum {
     BUILDING_LIGHTHOUSE = 155,
     BUILDING_MENU_STATUES = 156,
     BUILDING_MENU_GOV_RES = 157,
+    BUILDING_RUBBLE = 158,
```

Comment:
I don't like the idea of adding a building enum for rubble. Rubble is not a building and internally it is not treated as such, so it makes no logical sense to add it here. Check below for an alternative :+1: